src/providers/scicat-provider/dao/mongo-dao.ts

```diff
@@ -24,13 +25,20 @@ export class MongoConnector {
     this.dbName = process.env.DATABASE;
     this.collectionName = process.env.COLLECTION;
 
-    MongoClient.connect("mongodb://" + url, { useUnifiedTopology: true }, (err, client) => {
-      if (err) {
-        logger.error("failed to connect", err);
-        this.db = null;
-      }
-      this.db = client.db(this.dbName);
-    });
+     const  myUrl = "mongodb://"+url ;
```

Comment:
I would call it `mongoUrl`, I find it better to give variables names with a meaning

Reply:
Fixed